packages/remix-dev/cli/create.ts

```diff
@@ -51,9 +64,10 @@ export async function createApp({
   let projectDirIsCurrentDir = relativeProjectDir === "";
   if (!projectDirIsCurrentDir) {
     if (fse.existsSync(projectDir)) {
-      throw new Error(
+      console.error(
```

Comment:
Throwing known errors barfs out a call stack that isn't really useful to users. We know the problem, so let's just tell them and exit. They will open an issue if there's a problem and we can deal with the call stack. May want to pass the `--debug` flag here if they really want it.

Reply:
this technically doesn't have to be used only by the CLI which was the reason for throwing and having the CLI catch them and convert it to a log and process.exit

Reply:
Hmm, do we document this somewhere? I don't know that an export should constitute public API but we can chat about that. Happy to change it if we're catching them elsewhere, but currently if a user hits any error they get all the logs.